Example/PaymentSheet Example/PaymentSheetUITest/EmbeddedUITest.swift

```diff
@@ -14,11 +14,43 @@ class EmbeddedUITests: PaymentSheetUITestCase {
         settings.mode = .payment
         settings.integrationType = .deferred_csc
         settings.uiStyle = .embedded
+        settings.formSheetAction = .continue
         loadPlayground(app, settings)
         app.buttons["Present embedded payment element"].waitForExistenceAndTap()
-        // TODO: Test card form (see PaymentSheetVerticalUITests testUpdate)
-
-        // Selecting Alipay w/ deferred PaymentIntent...
+        
+        // Entering a card w/ deferred PaymentIntent...
+        app.buttons["Card"].waitForExistenceAndTap()
+        XCTAssertTrue(app.staticTexts["Add card"].waitForExistence(timeout: 10))
+        try! fillCardData(app, postalEnabled: true)
+        app.toolbars.buttons["Done"].tap()
+        XCTAssertTrue(app.buttons["Continue"].isEnabled)
+        app.buttons["Continue"].waitForExistenceAndTap()
+        XCTAssertTrue(app.staticTexts["Payment method"].waitForExistence(timeout: 10))
+        XCTAssertEqual(app.staticTexts["Payment method"].label, "•••• 4242")
+        
+        // ...and *updating* to a SetupIntent...
+        app.buttons.matching(identifier: "Setup").element(boundBy: 1).tap()
+        // ...(wait for it to finish updating)...
+        _ = app.buttons["Reload"].waitForExistence(timeout: 10)
```

Comment:
XCTAssertTrue on waitForExistence to get a better fail state in our tests. -- this should be applied to any time we "Reload"